<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkDialog</class>
 <widget class="QWidget" name="MarkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Mark</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QRadioButton" name="rBGo">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;GO (Kinematic)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QRadioButton" name="rBStop">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>STOP (Stati&amp;c)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Label2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Comment</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lblPositionMode">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Switch Positioning Mode</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="cBMarkerNameC">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Marker Name</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="lblPosition">
     <property name="text">
      <string>Lat/Lon/Height (deg/m)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="sBLongitude">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="decimals">
      <number>9</number>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QDoubleSpinBox" name="sBLatitude">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="decimals">
      <number>9</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QRadioButton" name="rBFix">
     <property name="text">
      <string>FIX (Fixed)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDoubleSpinBox" name="sBHeight">
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-500.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="btnPosition">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnKeyDlg">
     <property name="text">
      <string>?</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="Label1">
     <property name="text">
      <string>%r=001</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QComboBox" name="cBMarkerName">
     <property name="toolTip">
      <string/>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QLineEdit" name="lEMarkerComment">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QPushButton" name="btnCancel">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="btnOk">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cBMarkerNameC</tabstop>
  <tabstop>cBMarkerName</tabstop>
  <tabstop>lEMarkerComment</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
